dynamiqs/utils/utils.py

```diff
@@ -315,6 +462,29 @@ def lindbladian(H: Tensor, Ls: Tensor, rho: Tensor) -> Tensor:
 
     Returns:
         _(..., n, n)_ Resulting operator (it is not a density matrix).
+
+    Examples:
+        For example for a lossy quantum harmonic oscillator with Hamiltonian
+        $H=\omega a^\dag a$ and two jump operators $L_1=\sqrt{\kappa_1} a$ and
+        $L_\varphi=\sqrt{2\kappa_\varphi} a^\dag a$:
+        >>> import numpy as np
+        >>> a = dq.destroy(4)
+        >>> delta = 1.0
+        >>> kappa_1 = 0.1
+        >>> kappa_phi = 0.05
+        >>> H = delta * a.mH @ a
+        >>> L1 = np.sqrt(kappa_1) * a
+        >>> Lphi = np.sqrt(2 * kappa_phi) * a.mH @ a
+        >>> Ls = torch.stack([L1, Lphi])
+        >>> rho = dq.coherent_dm(4, 0.5)
+        >>> dq.lindbladian(H, Ls, rho)
+        tensor([[ 0.019+0.000j, -0.029+0.389j, -0.038+0.275j, -0.025+0.125j],
+                [-0.029-0.389j, -0.015+0.000j, -0.012+0.069j, -0.008+0.042j],
+                [-0.038-0.275j, -0.012-0.069j, -0.004+0.000j, -0.002+0.007j],
+                [-0.025-0.125j, -0.008-0.042j, -0.002-0.007j, -0.001+0.000j]])
```

Comment:
Is it necessary to introduce delta, kappa_1 and kappa_phi here? Why not just define directly:
```suggestion
>>> a = dq.destroy(4)
>>> H = a.mH @ a
>>> Ls = torch.stack([a, a.mH @ a])
>>> rho = dq.fock_dm(4, 1)
>>> dq.linbladian(H, Ls, rho)
```
for instance?